apps/server/src/connect/dataEndpoints.test.ts

```diff
@@ -26,9 +34,29 @@ const vcAccountsDataHandler = createAccountsDataHandler(true);
 const vcIdentityDataHandler = createIdentityDataHandler(true);
 const vcTransactionsDataHandler = createTransactionsDataHandler(true);
 
+const contextTestA = {
+  aggregator: TEST_EXAMPLE_A_AGGREGATOR_STRING,
+};
+let connectTestA: ConnectApi;
+
+const contextTestB = {
+  aggregator: TEST_EXAMPLE_B_AGGREGATOR_STRING,
+};
+let connectTestB: ConnectApi;
+
 describe("dataEndpoints", () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     jest.restoreAllMocks();
+
+    connectTestA = new ConnectApi({
```

Comment:
Why are these necessary? I see where they're passed in down below, but I don't think they're actually being used.

Reply:
If I don't do this, then the `req.connectApi` property isn't populated, and the code doesn't work.